yarn.lock

```diff
@@ -907,9 +907,9 @@
     "exec-sh" "^0.3.2"
     "minimist" "^1.2.0"
 
-"@digicat/components@file:/Users/MAddis/Documents/Code/Hydrogen/ui-component-library/packages/components":
+"@digicatapult/components@file:/Users/MAddis/Documents/Code/Hydrogen/ui-component-library/packages/components":
```

Comment:
Unsure why yarn is using the file syntax here. Feels wrong, please investigate.

Reply:
This is what other repos seem to be using.